### PR TITLE
add : added wishlist page

### PR DESCRIPTION
**Pull request by JuvitaSaini09**
Added wishlist page

snap shot

image

### CODE REVIEW

File: components/wishlist-page/wishlist-page.html [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+    <h1 class="heading-my-wishlist">My Wishlist &#x2764;</h1>
+
+    <section class="wishlist-card-container">
+
+        <div class="wishlist-card">
+            <div class="card card5 card7">
+                <section class="img-container">
+                    <img class="card-image" alt="card-image"
+                        src="https://images.unsplash.com/photo-1583743814966-8936f5b7be1a?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Nnx8dCUyMHNoaXJ0fGVufDB8fDB8fA%3D%3D&auto=format&fit=crop&w=400&q=60">
```
Comment (Adarsh-Patel-Dev) [inline components/wishlist-page/wishlist-page.html:42]: try to store photos inside  a folder(assets) in your files and then use it

File: components/wishlist-page/wishlist-page.html [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+
+    <footer class="footer-home">
+
+        <h2>Made by Juvita Saini with &#x2764;</h2>
+
+    </footer>
+
+</body>
+
```
Comment (Adarsh-Patel-Dev) [inline components/wishlist-page/wishlist-page.html:160]: Rest is looking good 
You can merged

**Review by Adarsh-Patel-Dev — APPROVED** [review]

File: components/wishlist-page/wishlist-page.html [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+                <h1><i class="fab fa-artstation"></i>Quartz</h1>
+            </div>
+
+            <nav class="nav">
+                <ul class="nav-links">
+                    <li><a href="/index.html">HOME</a></li>
+                    <li><a href="/components/product-page/product-page.html">PRODUCTS</a></li>
+                    <li><a href="/">ACCOUNT</a></li>
+                    <li><a href="/components/wishlist-page/wishlist-page.html"> <i class="fas fa-heart"></i></a></li>
+                    <li><a href="/components/cart-page/cart-page.html"><i class="fas fa-shopping-bag"></i></a></li>
```
Comment (rahulk31) [inline components/wishlist-page/wishlist-page.html:27]: Suggestion: To wrap the li tag with a tag

File: components/wishlist-page/wishlist-page.html (hunk 1 above)
Comment (gauravsinhaweb) [inline components/wishlist-page/wishlist-page.html:42]: use alt inside img.

File: components/wishlist-page/wishlist-page.html [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+                        </div>
+                    </div>
+
+                    <div class="cart-click-btns">
+                        <button class="btn-cart">Remove from wishlist</button><br>
+                        <button class="btn-cart">Add to cart</button>
+                    </div>
+
+
+
```
Comment (gauravsinhaweb) [inline components/wishlist-page/wishlist-page.html]: remove spacing

File: components/wishlist-page/wishlist-page.html [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+                    </div>
+
+                </section>
+            </div>
+        </div>
+
+    </section>
+
+
+
```
Comment (gauravsinhaweb) [inline components/wishlist-page/wishlist-page.html]: remove spacing.

**Review by gauravsinhaweb — APPROVED** [review]
Looks good! Good to merge after changes.

**Review by rahulk31 — APPROVED** [review]
Overall code looks well structured and can be merged. 👍

File: components/wishlist-page/wishlist-page.html (hunk 1 above)
Comment (JuvitaSaini09) [reply components/wishlist-page/wishlist-page.html:42]: I have used already,thnks for review

File: components/wishlist-page/wishlist-page.html (hunk 5 above)
Comment (JuvitaSaini09) [reply components/wishlist-page/wishlist-page.html]: I have made the change,thnks for reviewing

File: components/wishlist-page/wishlist-page.html (hunk 1 above)
Comment (theVivek-kumar) [reply components/wishlist-page/wishlist-page.html:42]: make a folder or image then use those image and rename those images

File: components/wishlist-page/wishlist-page.html [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+    <h1 class="heading-my-wishlist">My Wishlist &#x2764;</h1>
+
+    <section class="wishlist-card-container">
+
+        <div class="wishlist-card">
+            <div class="card card5 card7">
+                <section class="img-container">
+                    <img class="card-image" alt="card-image"
+                        src="https://images.unsplash.com/photo-1583743814966-8936f5b7be1a?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Nnx8dCUyMHNoaXJ0fGVufDB8fDB8fA%3D%3D&auto=format&fit=crop&w=400&q=60">
```
Comment (theVivek-kumar) [inline components/wishlist-page/wishlist-page.html:42]: make a folder or image then use those image and rename those images

File: components/wishlist-page/wishlist-page.html [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+                </section>
+            </div>
+        </div>
+
+        <div class="wishlist-card">
+            <div class="card card5 card7">
+                <section class="img-container">
+                    <img class="card-image" alt="card-image"
+                        src="https://images.unsplash.com/photo-1583743814966-8936f5b7be1a?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Nnx8dCUyMHNoaXJ0fGVufDB8fDB8fA%3D%3D&auto=format&fit=crop&w=400&q=60">
+                </section>
```
Comment (theVivek-kumar) [inline components/wishlist-page/wishlist-page.html:71]: here too use image folder

File: components/wishlist-page/wishlist-page.html [hunk 8]
```diff
@@ -0,0 +1,161 @@
...
+                <section class="text-container">
+                    <div class="card-text card8-text">
+                        <header>
+                            <h3>T-shirt</h3><button class="rate-btn"><span>&#9733</span>4.5</button>
+                        </header>
+                        <div class="card-info">
+                            <p>High comfort cotton</p>
+                            <div class="price">Rs. 300
+                                <span class="price-old">RS. 600</span>
+                            </div>
```
Comment (theVivek-kumar) [inline components/wishlist-page/wishlist-page.html:109]: use BEM convention or OCSS

File: components/wishlist-page/wishlist-page.html [hunk 9]
```diff
@@ -0,0 +1,161 @@
...
+        <div class="wishlist-card">
+            <div class="card card5 card7">
+                <section class="img-container">
+                    <img class="card-image" alt="card-image"
+                        src="https://images.unsplash.com/photo-1583743814966-8936f5b7be1a?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Nnx8dCUyMHNoaXJ0fGVufDB8fDB8fA%3D%3D&auto=format&fit=crop&w=400&q=60">
+                </section>
+                <section class="text-container">
+                    <div class="card-text card8-text">
+                        <header>
+                            <h3>T-shirt</h3><button class="rate-btn"><span>&#9733</span>4.5</button>
```
Comment (theVivek-kumar) [inline components/wishlist-page/wishlist-page.html:132]: structure is good use good css name

**Review by theVivek-kumar — COMMENTED** [review]
can be merged

**Pull request merged**